File: api/src/main/java/org/roylance/yaas/services/server/IUserDeviceService.java
package org.roylance.yaas.services.server;

import org.roylance.yaas.models.YaasModels;

import java.util.List;

public interface IUserDeviceService {
    void saveUserDevice(final YaasModels.UserDevice userDevice);
    List<YaasModels.UserDevice> getUserDevices(final YaasModels.User user);
}

File: capi/src/main/java/org/roylance/yaas/services/IUserDeviceService.kt
package org.roylance.yaas.services

interface IUserDeviceService {
	fun save(request: org.roylance.yaas.YaasModel.UIYaasRequest): org.roylance.yaas.YaasModel.UIYaasResponse
	fun all(request: org.roylance.yaas.YaasModel.UIYaasRequest): org.roylance.yaas.YaasModel.UIYaasResponse
}